src/components/Answers.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+function Answers({ answers, shouldShowCorrectAnswers = false }) {
+  const getCorrectOrWrongAnswerInfoList = answers.filter((answer) => (
```

Comment:
This is a constant, not a function call. Ideally should not start with get because this will not change and is a one time call.
Also, change variable name. You are filtering based on one parameter, right? so the answers in this should only be of one type, right?

Reply:
If it can be either, rename it filteredAnswerList or something like that.